ui/apps/app-center/src/components/pages/info-page.tsx

```diff
@@ -92,7 +95,7 @@ const InfoPage: React.FC<RootComponentProps> = props => {
           type="script-error"
           title={t('There was an error loading the integration info')}
           details={t('We cannot show this integration right now')}
-          devDetails={latestReleaseInfoReq.error}
+          devDetails={latestReleaseInfoReq.error as unknown as string}
```

Comment:
try to avoid typoe casting if possible. what is the original type of the error and why did it have to be changed?

Reply:
Error is unknown, but the devDetails is of type string. I will find another way to fix this